core/src/main/scala/quasar/sql/fixpoint.scala

```diff
@@ -62,8 +62,14 @@ object ArrayLiteral {
     ArrayLiteralF.unapply(obj.unFix)
 }
 
+/**
+ * Represents the wildcard in a select projection
+ * For instance:
+ *  "select foo.* from example" => ...(Splice(Some(Ident("foo"))))...
+ *  "select * from example"     => ...(Splice(None))...
+ */
```

Comment:
Scala doc comments look like:
```scala
/** Represents the wildcard in a select projection
  * For instance:
  *  "select foo.* from example" => ...(Splice(Some(Ident("foo"))))...
  *  "select * from example"     => ...(Splice(None))...
  */
```
and by “too”, I just meant “in addition to the other changes I mentioned,” since this is below the hassle threshold if it were the only change to be made to the PR.